recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 5275bb04f4863a13516b2f39392ac5e272f5e1bb8057b18aec1c9b79d73d8fb2
 
 build:
-  number: 1000
+  number: 1001
```

Comment:
Why?

Reply:
oh, I thought we usually bunp build after rerender, since rerender could have changed the dependencies.  wrong?

Reply:
Sure, but in this case they are not. gmp doesn't have other libraries depending on it except for the compiler runtimes

Reply:
In general (maybe not here), can rerender change compiler version, or the system libs in the linux-anvil docker image?

Reply:
Sure. But those will be major infrastructure changes done through the bot and will not happen directly using a rerender until the migration is over.

Reply:
Ok.  Then, maybe it shouldn't be re-rendered at all?  Won't it be an error to try to upload a package with the same name and build number as already exists in the conda-forge channel?

Reply:
If the built package already exists, upload script will simply do nothing.